app/lib/features/tasks/pages/task_list_details_page.dart

```diff
@@ -162,7 +162,7 @@ class _TaskListPageState extends ConsumerState<TaskListDetailPage> {
   Widget _buildTaskListInner(TaskList? taskListData) {
     return SingleChildScrollView(
       child: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 20.0),
+        padding: const EdgeInsets.symmetric(horizontal: 8.0),
```

Comment:
what is this about? I didn't see this in the promo video...?

Reply:
Ahh, I was play with TaskList Details padding-spacing improvements. Here with [this commit](https://github.com/acterglobal/a3/pull/2464/commits/de1c7fa4d7f9148702d8dede7acb7b3da647ebc2) I completed improvements. Also sharing screenshot for reference.

![TaskList](https://github.com/user-attachments/assets/631dde4b-7be9-43c5-b64b-8d3381cb4c9a)